fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/minecraft/FluidIds.java

```diff
@@ -0,0 +1,99 @@
+package net.fabricmc.fabric.api.fluids.minecraft;
+
+import static net.minecraft.util.registry.Registry.FLUID;
+
+import java.util.Objects;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.Fluids;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+/**
+ * the ids for all vanilla fluids
+ */
+public class FluidIds {
+	public static final Identifier WATER = get(Fluids.WATER);
+	public static final Identifier LAVA = get(Fluids.LAVA);
+	public static final Identifier EMPTY = get(Fluids.EMPTY);
+	/**
+	 * found in honey bottles
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier HONEY = new Identifier("honey");
+	/**
+	 * found in splash/lingering/normal potion bottles
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier POTION = new Identifier("potion");
+	/**
+	 * found in mushroom stew
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier MUSHROOM_STEW = new Identifier("mushroom_stew");
+	/**
+	 * found in experience bottles
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier EXPERIENCE = new Identifier("enchanting");
+	/**
+	 * found in suspicious stew
+	 * todo fluid properties, do suspicious stew store effects in NBT?
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier SUSPICIOUS_STEW = new Identifier("suspicious_stew");
+	/**
+	 * found in bottles of dragon's breath
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier DRAGONS_BREATH = new Identifier("dragon_breath");
+	/**
+	 * found in buckets of milk
+	 *
+	 * @see #shouldAdd(Identifier)
+	 */
+	public static final Identifier MILK = new Identifier("milk");
```

Comment:
All of this should go as per my other comments.

Reply:
gone